### PR TITLE
Replace CurrentBlock() with CurrentBlockHeight()

### PR DESCRIPTION
**Pull request by forjoin92**
Replace "CurrentBlock()" with "CurrentBlockHeight() "

### CODE REVIEW

File: rpc/seele_rpc.go [hunk 1]
```diff
@@ -171,8 +153,13 @@ func (rpc *SeeleRPC) GetPeersInfo() (result []PeerInfo, err error) {
 	//     ]
 	//   shard:2
 	// ]
+	return getPeerInfos(rpcPeerInfos), nil
+}
+
+// getPeerInfos parse peer informations map to PeerInfo
```
Comment (WTer) [inline rpc/seele_rpc.go]: informations ->information

Comment (forjoin92) [reply rpc/seele_rpc.go]: I have fixex the typo error.

File: rpc/seele_rpc.go [hunk 2]
```diff
@@ -213,20 +200,26 @@ func (rpc *SeeleRPC) GetBalance(address string) (int64, error) {
...
+
+// getBalance parse balance informations map to int64
+func getBalance(balanceMp map[string]interface{}, account string) (int64, error) {
+	retAccount := balanceMp["Account"].(string)
+	if retAccount != account {
+		return 0, fmt.Errorf("expected balance '%s', actually '%s'", account, retAccount)
 	}
-	balance := int64(result["Balance"].(float64))
-	return balance, nil
+	return int64(balanceMp["Balance"].(float64)), nil
```
Comment (WTer) [inline rpc/seele_rpc.go]: will data be lost converting float64 to int64?

File: rpc/seele_rpc_test.go [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+*  @file
+*  @copyright defined in scan-api/LICENSE
+ */
+
+package rpc
+
+import (
+	"encoding/json"
+	"math/big"
+	"testing"
```
Comment (WTer) [inline rpc/seele_rpc_test.go]: refer to go-seele to introduce assert

Comment (forjoin92) [reply rpc/seele_rpc_test.go]: done

File: rpc/seele_rpc.go (hunk 2 above)
Comment (forjoin92) [reply rpc/seele_rpc.go]: no, if a value is float64, I will not use int64 to receive it

**Pull request merged**